packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/index.tsx

```diff
@@ -78,9 +78,9 @@ const SeriesTab: FC<Props> = ({ items }) => {
             const reorderedSeries = reorderedSeriesGroups.reduce<Series[]>(
                 (acc, seriesGroup) => [
                     ...acc,
-                    ...seriesGroup.value.map((s, i) => ({
+                    ...seriesGroup.value.map((s) => ({
                         ...s,
-                        color: getSeriesColor(s, i),
+                        color: getSeriesColor(s),
```

Comment:
We no longer need the series index to get a series color, since we're precalculating `seriesId->color` in the parent context.